src/acceptance-test/java/org/zalando/nakadi/repository/db/StorageDbRepositoryTest.java

```diff
@@ -64,15 +61,15 @@ public void testStorageCreated() throws Exception {
     }
 
     @Test
-    public void testStorageOrdered() throws Exception {
+    public void testStorageOrdered() {
         final String namePrefix = randomValidStringOfLength(31);
 
         final Storage storage2 = repository.createStorage(
-                createStorage(namePrefix + "2", "exaddress", 8181, "address1", "path3"));
+                createStorage(namePrefix + "2", ZookeeperConnection.valueOf("exhibitor://exaddress:8181/path")));
         final Storage storage1 = repository.createStorage(
-                createStorage(namePrefix + "1", "exaddress", 8181, "address2", "path2"));
+                createStorage(namePrefix + "1", ZookeeperConnection.valueOf("exhibitor://exaddress:8181/path")));
         final Storage storage3 = repository.createStorage(
-                createStorage(namePrefix + "3", "exaddress", 8181, "address3", "path1"));
+                createStorage(namePrefix + "3", ZookeeperConnection.valueOf("exhibitor://exaddress:8181/path")));
```

Comment:
As I see there were 3 storages here with different addresses and different paths. But you changed it in a way that all storages have the same address/path.
How can you explain that?

Reply:
Done